src/toil/lib/plugins.py

```diff
@@ -0,0 +1,66 @@
+import importlib
```

Comment:
We probably want to have the copyright header at the top here like we have in the other Toil source files, and also maybe a note (module-level docstring?) about what this file is for.